app/controllers/staff/training_sessions_controller.rb

```diff
@@ -0,0 +1,130 @@
+class Staff::TrainingSessionsController < StaffAreaController
+  before_action :load_user, only: [:show, :edit, :update]
+
+  layout 'staff_area'
+
+  def index
```

Comment:
Do you mean replacing it later and leaving it as is or cleaning up what's here and adding it later?